Source/OxyPlot/Axes/Axis.cs

```diff
@@ -722,10 +722,21 @@ public virtual void CoerceActualMaxMin()
             var range = this.ActualMaximum - this.ActualMinimum;
             if (range < this.MinimumRange)
             {
-                var average = (this.ActualMaximum + this.ActualMinimum) * 0.5;
-                var delta = this.MinimumRange / 2;
-                this.ActualMinimum = average - delta;
-                this.ActualMaximum = average + delta;
+                if (!double.IsNaN(this.Minimum))
+                {
+                    this.ActualMaximum = this.Minimum + this.MinimumRange;
```

Comment:
to keep the current behavior you could use the current code `if (!double.IsNaN(this.Minimum) && !double.IsNaN(this.Maximum))`, and use the new code in the two other cases